app/routes/events/components/EventFooter.tsx

```diff
@@ -19,10 +19,12 @@ const icalTypes = [
   },
 ];
 
-const getIcalUrl = (icalToken, icalType) =>
+type IcalType = 'events' | 'registrations' | 'personal';
+
+export const getIcalUrl = (icalToken: string, icalType: IcalType) =>
```

Comment:
```suggestion
export const getIcalUrl = (icalToken: IcalToken, icalType: IcalType) =>
```
`IcalToken` can be imported from `app/models.ts`